### DISCUSSION TITLE
faster whisper xxx

### DISCUSSION THREAD

**Discussion by Policing2024** [SubtitleEdit/subtitleedit#8289] [General]
![Screenshot_20240501-171953.jpg](https://github.com/SubtitleEdit/subtitleedit/assets/134338358/77b76092-2686-40a3-9654-eafbe13db9b2)

how to install alt vad
2. how to run faster whisper with cuda
pls

**Comment by Policing2024** [comment]
I have

https://github.com/ggerganov/whisper.cpp/releases/download/v1.5.4/whisper-cublas-12.2.0-bin-x64.zip

https://huggingface.co/ggerganov/whisper.cpp/resolve/main/ggml-large-v3-q5_0.bin
!

how to use it with subtitle edit pls

**Comment by ClaireCJS** [comment]
faster-whisper-XXX will mention if you are using CUDA

See attached screenshot's double-height text and how it says "```CUDA```"

[i have a postprocessor that changes the appearance of things]

![image](https://github.com/user-attachments/assets/0e8606bb-7a69-4451-aabb-e44e45202f22)
